app/src/main/res/values/strings.xml

```diff
@@ -93,12 +93,22 @@
         for story templates does not contain usable templates.  Select the appropriate
         \'&lt;font color=\"green\">&lt;b>SP Templates&lt;/b>&lt;/font>\' folder - normally found on the
         &lt;font color=\"green\">&lt;b>SD card&lt;/b>&lt;/font>.</string>
-    <string name="use_internal_demo_or_select_template_folder">Would you like to use the internal demo or select the appropriate
-        \'&lt;font color=\"green\">&lt;b>SP Templates&lt;/b>&lt;/font>\' folder -
-        normally found on the &lt;b>&lt;font color=\"green\">SD card&lt;/font>&lt;/b>?</string>
-    <string name="workspace_selection_help">On the next screen - &lt;br/&gt;SELECT the appropriate
-        \'&lt;font color=\"green\">&lt;b>SP Templates&lt;/b>&lt;/font>\' folder -
-        normally found on the &lt;b>&lt;font color=\"green\">SD card&lt;/font>&lt;/b>.</string>
+    <string name="welcome_screen_select_template_folder">
+        Story Producer is different from typical apps.
+        You must first create a folder on your SD card (preferred) where the story template files must be downloaded by you.
+        &lt;font color=\"gray\">(If your SD card has already been prepared, skip down to \"Select \'SP Templates\' Folder\")&lt;/font>&lt;br>&lt;br>
+        To prepare your SD card&#8230;&lt;br>
+        \t\tGo &lt;a href="https://docs.google.com/document/d/1Jw73VlLmIyJgoFM3EOLyqGZyO50pOqXiSmsuLuq_u34/edit">HERE&lt;/a> for detailed instructions&lt;br>
+        \t\tOR follow these summary instructions:&lt;br>
+        1. Use Google Drive and download story templates from &lt;a href="https://drive.google.com/drive/folders/1CxpggJUJ6QPnNgb3Veh9r7SWiLfPKCDj?usp=sharing">HERE&lt;/a>&lt;br>
+        2. Use a Files app to create a folder on the SD card, and name it &lt;font color=\"green\">SP Templates&lt;/font>&lt;br>
+        3. With the Files app, Move or Copy story template(s) to the &lt;font color=\"green\">SP Templates&lt;/font> folder&lt;br>&lt;br>
+        After you have prepared the \'SP Templates\' folder, proceed:
+    </string>
+    <string name="workspace_selection_help">
+        On the next screen is a \'folder picker\'.&lt;br>
+        Navigate to the &lt;font color=\"green\">SP Templates&lt;/font> folder, then tap &lt;b>SELECT&lt;/b> at the bottom.
```

Comment:
Please add bold to the green string  SP Templates
Also a new request - add this parenthetical:   folder (on the &lt;b>&lt;font color=\"green\">SD card&lt;/font>&lt;/b>)
Actually, I'd like to revise several more items in this dailog - please update per the google doc. Thanks!

Reply:
I can do this!